include/k4arecord/playback.h

```diff
@@ -134,6 +134,203 @@ K4ARECORD_EXPORT k4a_result_t k4a_playback_get_calibration(k4a_playback_t playba
 K4ARECORD_EXPORT k4a_result_t k4a_playback_get_record_configuration(k4a_playback_t playback_handle,
                                                                     k4a_record_configuration_t *config);
 
+/** Checks whether a track with the given track name exists in the playback file.
+ *
+ * \param playback_handle
+ * Handle obtained by k4a_playback_open().
+ *
+ * \param track_name
+ * The track name to be checked to see whether it exists or not.
+ *
+ * \returns true if the track exists.
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">playback.h (include k4arecord/playback.h)</requirement>
+ *   <requirement name="Library">k4arecord.lib</requirement>
+ *   <requirement name="DLL">k4arecord.dll</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+K4ARECORD_EXPORT bool k4a_playback_check_track_exists(k4a_playback_t playback_handle, const char *track_name);
+
+/** Get the number of tracks in a playback file.
+ *
+ * \param playback_handle
+ * Handle obtained by k4a_playback_open().
+ *
+ * \returns the number of tracks in the playback file.
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">playback.h (include k4arecord/playback.h)</requirement>
+ *   <requirement name="Library">k4arecord.lib</requirement>
+ *   <requirement name="DLL">k4arecord.dll</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+K4ARECORD_EXPORT size_t k4a_playback_get_track_count(k4a_playback_t playback_handle);
+
+/** Gets the name of a track at a specific index.
+ *
+ * \param playback_handle
+ * Handle obtained by k4a_playback_open().
+ *
+ * \param track_index
+ * The index of the track to read the name form.
+ *
+ * \param track_name
+ * Location to write the track name. This will be a UTF8 null terminated string. If a NULL buffer is specified, \p
```

Comment:
>\p [](start = 112, length = 2)

my doxygen-fu is weak so this might be fine, but it looks like the line wrap separated this annotation from the variable it refers to - if this actually works in doxygen it's fine, but I wouldn't be surprised if this doesn't work, might want to move to the next line

Reply:
It looks like doxygen figures this out just fine ([See Here](https://microsoft.github.io/Azure-Kinect-Sensor-SDK/feature/support-custom-track-record-playback/group___functions_gaaa6e6d931691c5fde0db0a831286109f.html#gaaa6e6d931691c5fde0db0a831286109f)), but I'll change this anyway just so it's a bit more readable in the source code.